build.gradle

```diff
@@ -369,6 +331,49 @@ publishing {
     }
 }
 
+afterEvaluate {
```

Comment:
Let's add some comments about a) why we are doing this here now, and b) how to revert if needed/in case we move to the non-software model plugins.

Reply:
I added some comments.